apps/dsb-message-broker/src/channel/channel.controller.ts

```diff
@@ -61,19 +64,48 @@ export class ChannelController {
         }
     }
 
+    @Patch()
+    @Role('user')
```

Comment:
shouldn't this be `channelCreator`?

Reply:
admin did checking for modification is additional to this user role checking.
https://github.com/energywebfoundation/dsb/blob/a038e38e904977c09743c5d6c0957d6bc907ccec/apps/dsb-message-broker/src/channel/channel.service.ts#L35

Reply:
yeah I realised an admin might not be a `channelCreator`. But might it need to be both? If the original admin only has `channelCreator`?

Reply:
in channel creation if admins filed was missing then mb puts the creators did in admins array. but if was there then it will be restricted to that regardless of the creator.

Reply:
I meant for just having access to this role - like for example if I create a channel and want to see it after creation, I cannot fetch /channel/pubsub or /channel/{fqcn} because I don't have the `user` role. So it would be better to take an array of roles instead of a single role.

Reply:
Not a big deal right now, just something to consider.